src/components/Navigation.js

```diff
@@ -1,18 +1,20 @@
 import React from 'react';
 
+import { Link } from 'react-router-dom';
+
 const Navigation = () => {
   return (
     <div>
       <div className="App">
         <h1>React Router Mini</h1>
         <div>
-          <a href="">Home</a>
+          <Link to="/">Home</Link>
         </div>
         <div>
-          <a href="">About</a>
+          <Link to="/about">About</Link>
         </div>
         <div>
-          <a href="">Contact</a>
+          <Link to="/contact">Contact</Link>
```

Comment:
What is the difference between a `Link` component and an `a` tag? What would happen if we use one instead of the other?